Makefile

```diff
@@ -39,6 +45,15 @@ PROJECT_NAME:=NodeMCU
 # conventions for sized integers across the various components.
 BASIC_TYPES=-Du32_t=uint32_t -Du16_t=uint16_t -Du8_t=uint8_t -Ds32_t=int32_t -Ds16_t=int16_t -Duint32=uint32_t -Duint16=uint16_t -Duint8=uint8_t -Dsint32=int32_t -Dsint16=int16_t -Dsint8=int8_t
 
+$(IDF_PATH)/make/project.mk:
+	@echo "esp32-esp-idf isn't properly installed. Re-install and re-run make:"
```

Comment:
"esp32-esp-idf" -> no such thing, use "ESP-IDF" instead.

"Re-install and re-run make" sounds like a request for the user to do this. Instead, make is doing that itself -> rephrase.